README.md

```diff
@@ -111,7 +111,7 @@ Visit `http://localhost:8000` in your browser, and sign in with your system user
 
 _Note_: To allow multiple users to sign in to the server, you will need to
 run the `jupyterhub` command as a _privileged user_, such as root.
-The [wiki](https://github.com/jupyterhub/jupyterhub/wiki/Using-sudo-to-run-JupyterHub-without-root-privileges)
+The [Run JupyterHub without root privileges using sudo](https://jupyterhub.readthedocs.io/en/latest/howto/configuration/config-sudo.html)
```

Comment:
We can refer to the documentation, rather than giving the page title by name here
```suggestion
The [documentation](https://jupyterhub.readthedocs.io/en/latest/howto/configuration/config-sudo.html)
```